tests/spread/pack/grammar/imagecraft.yaml

```diff
@@ -0,0 +1,40 @@
+name: grammar-project
+version: "0.1"
+base: bare
+build-base: ubuntu@24.04
+
+platforms:
+  amd64-generic:
+    build-on: [amd64]
+    build-for: [amd64]
+  raspi-arm64:
+    build-on: [amd64, arm64]
+    build-for: [arm64]
+
+filesystems:
+  default:
+    - mount: /
+      device: (volume/disk/rootfs)
+
+parts:
+  my-part:
+    plugin: nil
+
+volumes:
+  disk:
+    schema: gpt
+    structure:
```

Comment:
This is a contrived example, I was having trouble getting a more real-world scenario to build.